sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -20,7 +20,7 @@ import groovy.json.JsonOutput
 
 plugins { id 'org.apache.beam.module' }
 applyJavaNature(
-
+  //enableStrictDependencies:true,
```

Comment:
@ibzib to your question from the parent PR:

> I ran into this as well. I was able to avoid it temporarily by downgrading from Gradle 6.8 to 6.7, however we will need to figure out a better solution. The first step is to figure out where the :runners:core-java dependency is coming from, since it isn't explicitly declared here.

running ` ../../../../gradlew -q dependencies` shows that :runners:java-core comes from :sdks:java:extensions:google-cloud-platform-core

Reply:
I figured out that this was a bug in the plugin: https://github.com/wfhartford/gradle-dependency-analyze/issues/125

This bug only occurs when there is a strict dependency violation.

You can see the real violation when doing your local testing by reverting https://github.com/apache/beam/commit/6763c2175c27436e3a82e06725fd2754fd71ad48.

Reply:
@ibzib thanks for the tip. the only violation I see is that runners:core-java was usedUndeclared. The only fix I see is to add that explicitly.